pkgs/applications/misc/synergy/default.nix

```diff
@@ -79,8 +79,8 @@ stdenv.mkDerivation rec {
     Cocoa
     CoreServices
     ScreenSaver
-  ] ++ lib.optionals (stdenv.isDarwin && darwin.apple_sdk.frameworks ? UserNotifications) [
-    darwin.apple_sdk.frameworks.UserNotifications
+  ] ++ lib.optionals (stdenv.isDarwin && !isNull UserNotifications) [
```

Comment:
```suggestion
  ] ++ lib.optionals stdenv.isDarwin [
```

Reply:
Done in commit 7bf7cb5a576fc72ccca5bb217fbb625e0c1a27af.